core/src/main/java/inetsoft/web/portal/controller/DataTreeChangeController.java

```diff
@@ -44,6 +47,7 @@ public DataTreeChangeController(AssetRepository assetRepository,
    {
       this.assetRepository = assetRepository;
       this.messagingTemplate = messagingTemplate;
+      this.orgManager = OrganizationManager.getInstance();
```

Comment:
In general, if you're not injecting a dependency, but getting it with the singleton getter, do it at the point of access instead of assigning it to a member property in the constructor.